test.js

```diff
@@ -0,0 +1,14 @@
+Config = require('./Config');
+
+const config = new Config('12345', 1000).connect();
+const sdkClient = config.client;
+
```

Comment:
I suggest adding `sdkClient.addContext({userId: "abadoicuvoij"}` or something like that, since you'll get an error otherwise (unless that's what you were going for).

Reply:
What error are you referring to?